storage/storage_dest.go

```diff
@@ -783,11 +778,15 @@ func (s *storageImageDestination) createNewLayer(index int, layerDigest digest.D
 		options := &graphdriver.ApplyDiffWithDifferOpts{
```

Comment:
(Absolutely non-blocking: I wouldn’t mind inlining single-use `options` and `args` users into their users.

This works fine, and it is a smaller patch.)

Reply:
fixed